code/drasil-example/pdcontroller/lib/Drasil/PDController/Body.hs

```diff
@@ -146,13 +148,13 @@ symbMap = cdb (map dqdWr physicscon ++ symbolsAll ++ [dqdWr mass, dqdWr posInf,
   genDefns
   theoreticalModels
   conceptInstances
-  labelledContent
+  labelledContentWithInputs
   allRefs
   citations
 
 -- | Holds all references and links used in the document.
 allRefs :: [Reference]
-allRefs = [externalLinkRef]
+allRefs = externalLinkRef : SRS.sectionReferences ++ map ref labelledContentWithInputs
```

Comment:
Because this change, which is likely also good, will also change `stable`. We want to review these as separately as possible.